scenes/ui_elements/input_hints/throw_hint.tscn

```diff
@@ -1,12 +1,13 @@
-[gd_scene load_steps=8 format=3 uid="uid://cdwf2nmeqns31"]
+[gd_scene load_steps=9 format=3 uid="uid://cdwf2nmeqns31"]
 
 [ext_resource type="Theme" uid="uid://cvitou84ni7qe" path="res://scenes/ui_elements/components/theme.tres" id="1_hvmxk"]
 [ext_resource type="Texture2D" uid="uid://d4b0kmnbktqeh" path="res://assets/third_party/inputs/Xbox Series/XboxSeriesX_A.png" id="2_cgyew"]
 [ext_resource type="Script" uid="uid://cbj0406q05dly" path="res://scenes/game_elements/props/hint/input_key/interact_input.gd" id="3_g0g4i"]
 [ext_resource type="Texture2D" uid="uid://p8kty3uxifxu" path="res://assets/third_party/inputs/keyboard-and-mouse/Dark/Space_Key_Dark.png" id="4_81p17"]
-[ext_resource type="Texture2D" uid="uid://dm4a8fn88sjf5" path="res://assets/third_party/inputs/PS5/PS5_Cross.png" id="5_vu6nt"]
-[ext_resource type="Texture2D" uid="uid://bc2ktlkypid78" path="res://assets/third_party/inputs/Switch/Switch_A.png" id="6_trb8c"]
-[ext_resource type="Texture2D" uid="uid://dul7b6elfhcm0" path="res://assets/third_party/inputs/Steam Deck/SteamDeck_A.png" id="7_3bs2i"]
+[ext_resource type="Texture2D" uid="uid://8vjuvwg6uvkq" path="res://assets/third_party/inputs/Xbox Series/XboxSeriesX_RT.png" id="5_cgyew"]
+[ext_resource type="Texture2D" uid="uid://bersmjts8ia2e" path="res://assets/third_party/inputs/PS5/PS5_R2.png" id="6_g0g4i"]
+[ext_resource type="Texture2D" uid="uid://22x1c6271exs" path="res://assets/third_party/inputs/Switch/Switch_RT.png" id="7_81p17"]
```

Comment:
I believe you have a Switch so I'll take your word for it that this is the right label for the button! :)